build/Makefile.tcmalloc

```diff
@@ -3,6 +3,10 @@ ifeq ($(UNAME),FreeBSD)
 EXTRA_LDFLAGS="-lexecinfo"
 endif
 
+ifeq ($(UNAME),Linux)
+EXTRA_LDFLAGS="-lunwind"
```

Comment:
You _may_ end up want this on FreeBSD too, certainly the configure-ct script adds `-lunwind` to the CT `configure` run for FreeBSD.